book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```

Comment:
- [ ] Since we want to be able to also set the `rentals`, kindly remove it from the attr_reader and it to attr_accessor.

  ```
  attr_accessor :title, :author, :rentals
  ```

Reply:
With all due respect, @julie-ify, I disagree, `:rentals` is a mutable object that has its own methods to handle data internally, if we move it to `attr_accessor`, it can be overridden which should never happen.